assets/js/cf7-paypal-admin.js

```diff
@@ -0,0 +1,62 @@
+document.addEventListener("DOMContentLoaded", function(){
```

Comment:
Improve readability as below.
```suggestion
document.addEventListener( 'DOMContentLoaded', function() {
```